addons/web/static/src/core/popover/popover.js

```diff
@@ -6,15 +6,42 @@ const { Component } = owl;
 
 export class Popover extends Component {
     setup() {
-        usePosition(this.props.target, "popper", {
-            margin: 16,
+        usePosition(this.props.target, {
             onPositioned: this.onPositioned,
             position: this.props.position,
         });
     }
     onPositioned(el, { direction, variant }) {
         const position = `${direction[0]}${variant[0]}`;
-        el.classList.add(`o-popper-position--${position}`);
+        el.classList.add(`bs-popover-${direction}`, `o-popover--${position}`);
+        if(el.childNodes[0].classList.contains('arrow')) {
+            switch(position) {
+                case 'tm': // top-middle
+                case 'bm': // bottom-middle
+                    el.childNodes[0].classList.add('start-0', 'end-0', 'mx-auto');
+                    break;
+                case 'lm': // left-middle
+                case 'rm': // right-middle
+                    el.childNodes[0].classList.add('top-0', 'bottom-0', 'my-auto');
+                    break;
+                case 'ts': // top-start
+                case 'bs': // bottom-start
+                    el.childNodes[0].classList.add('end-auto');
+                    break;
+                case 'te': // top-end
+                case 'be': // bottom-end
+                    el.childNodes[0].classList.add('start-auto');
+                    break;
+                case 'ls': // left-start
+                case 'rs': // right-start
+                    el.childNodes[0].classList.add('bottom-auto');
+                    break;
+                case 'le': // left-end
+                case 're': // right-end
+                    el.childNodes[0].classList.add('top-auto');
```

Comment:
You introduced a bunch of linting issue in this file.
Please use double quotes instead of single quotes.

All the tooling configurations for the `web[_*]` addons are available in the `community/addons/web/tooling` folder.

If you use VSCode you can enable it like this:
```shell
$ cd your_odoo_workspace
$ cd community # the community repo should have this name, the enterprise repo should be located next to it and it should have the name "enterprise"
$ ./addons/web/tooling/enable.sh
```

Please feel free to ping me if you want some help setting up this in your IDE.